### DISCUSSION TITLE
Tagline for mosaic

### DISCUSSION THREAD

**Discussion by h3nill** [zellij-org/zellij#104] [General]
IMO this does not have to be formal - describe what you are, kind of thing. So any kind of ideas is most welcome. 

For reference, some that I like:

[bat](https://github.com/sharkdp/bat): cat(1) clone with wings
[ImHex](https://github.com/WerWolv/ImHex): A Hex Editor for Reverse Engineers, Programmers and people that value their eye sight when working at 3 AM. 
[gitea](https://github.com/go-gitea/gitea): Git with a cup of tea, painless self-hosted git service 
[hostyoself](https://github.com/schollz/hostyoself): Host yo' self from your browser, your phone, your toaster. 

Maybe something on the lines of:
> Mosaic is productivity platform for terminal lovers.

**Comment by imsnif** [comment]
Hey @henil, I was recently reminded of this quote which I really like and I feel describes the vision of Mosaic. What do you think?

_"I’m talking about is the death of the 'application'. It’s time we simply got rid of them. Getting hold of the tools I need should be as simple as pasting them."
-- Douglas Adams, "A Salmon of Doubt"_

**Comment by imsnif** [comment]
Another one of his from the same chapter which I like:

_"We should not have to be tyrannised by application designers who don't know the first thing about how actual people do their actual work. We should be able to just pick up the application bits we like and paste them [into our workspace]."
-- Douglas Adams, "A Salmon of Doubt"_

**Comment by imsnif** [comment]
Maybe combine them both?

_"I’m talking about is the death of the 'application'. It’s time we simply got rid of them. [...] We should not have to be tyrannised by application designers who don't know the first thing about how actual people do their actual work. We should be able to just pick up the bits we like and paste them into our workspace."
-- Douglas Adams, "A Salmon of Doubt"_

**Reply by h3nill** [reply]
I like the essence of what the quote is trying to convey, in a broad sense it captures the vision of Mosaic.

But I think for a tagline the whole quote is quite big, It would be great if we could somehow take the part `getting hold of the tools I need should be as simple as pasting them` and/or ` should be able to just pick up the bits we like and paste them into our workspace` and compress it to a single line.

Something like, `Terminal workspaces should be as easy as copy-pasting`, but the word *easy* is relative like I find `vim` much more *easy* to use than `vscode`, but someone else might not.

**Comment by qballer** [comment]
Copy-paste terminal applications
